pkg/config/stepmeta_test.go

```diff
@@ -198,7 +198,7 @@ func TestGetParameterFilters(t *testing.T) {
 					}
 				}
 			})
-			t.Run("Env", func(t *testing.T) {
+			t.Run("SetEnv", func(t *testing.T) {
```

Comment:
Not sure if we should rename here since it is checking an Env (related to the stepmetadata) and not something like `SetEnv` (related to the os env)

Reply:
I refactored that by mistake with rename all.
Will restore them.